app/src/main/res/values/styles.xml

```diff
@@ -62,4 +62,13 @@
         <item name="layout_constraintTop_toTopOf">parent</item>
         <item name="titleTextColor">@color/app_bar_text_color</item>
     </style>
+
+    <style name="FrameLayout">
```

Comment:
How about "ContentFrameLayout"  for naming?

Reply:
I'll rename so its name is clearer.
Thank you for your suggestion.